python/tests/unit/memory/test_weaviate_memory_store.py

```diff
@@ -0,0 +1,290 @@
+import numpy as np
+import numpy.testing as npt
+import pytest
+
+from semantic_kernel.memory import weaviate_memory_store
+from semantic_kernel.memory.memory_record import MemoryRecord
+
+
+@pytest.fixture
+def documents():
+    records = []
+
+    records.append(
+        MemoryRecord.local_record(
+            "1",
+            "The quick brown fox jumps over the lazy dog.",
+            "A classic pangram.",
+            np.array([0.1, 0.1]),
+        )
+    )
+    records.append(
+        MemoryRecord.local_record(
+            "2",
+            "The five boxing wizards jump quickly.",
+            "Another popular pangram.",
+            np.array([0.1, 0.11]),
+        )
+    )
+    records.append(
+        MemoryRecord.local_record(
+            "3",
+            "Pack my box with five dozen liquor jugs.",
+            "A useful pangram.",
+            np.array([0.11, 0.1]),
+        )
+    )
+
+    records.append(
+        MemoryRecord.local_record(
+            "4",
+            "Lorem ipsum dolor sit amet.",
+            "A common placeholder text.",
+            np.array([-10, -10]),
+        )
+    )
+    records.append(
+        MemoryRecord.local_record(
+            "5",
+            "Etiam faucibus orci vitae lacus pellentesque.",
+            "A Latin text.",
+            np.array([-10.1, -10.2]),
+        )
+    )
+
+    yield records
+
+
+@pytest.fixture
+def memory_store():
+    config = weaviate_memory_store.WeaviateConfig(use_embed=True)
```

Comment:
Will these tests run successfully with no URL provided to the weaviate config? 

Also, I think this should be an integration test.

Reply:
yes, it will run with no url because it is running weaviate in [embedded mode](https://weaviate.io/developers/weaviate/installation/embedded) which means it will download the binaries.

I will move this file to the integration tests folder